utilities/constants.py

```diff
@@ -110,10 +110,53 @@ class Cdi:
             QCOW2_IMG = "cirros-qcow2.img"
             DIR = f"{BASE_IMAGES_DIR}/cdi-test-images"
 
+    class S390X:  # noqa: N801
+        class Cirros:
+            RAW_IMG = "Fedora-Cloud-Base-Generic-41-1.4.s390x.raw"
+            RAW_IMG_GZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.raw.gz"
+            RAW_IMG_XZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.raw.xz"
+            QCOW2_IMG = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2"
+            QCOW2_IMG_GZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2.gz"
+            QCOW2_IMG_XZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2.xz"
+            DISK_DEMO = "fedora-cloud-registry-disk-demo"
+            DIR = f"{BASE_IMAGES_DIR}/fedora-images"
+            DEFAULT_DV_SIZE = "10Gi"
+            DEFAULT_MEMORY_SIZE = "1Gi"
+
+        class Rhel:
+            RHEL9_5_IMG = "rhel-95-s390x.qcow2"
+            RHEL9_5_ARM64_IMG = "rhel-95-aarch64.qcow2"
```

Comment:
this is arm64 image. need to create and move it unget Arm64 class (can be done in separate pr)

Reply:
done